frontend/lib/wallets/Coinbase.ts

```diff
@@ -21,13 +22,19 @@ export default class Coinbase extends AbstractWallet {
         }
     }
 
-    getProvider(): ethers.providers.JsonRpcProvider {
-        const rpcUrl = process.env.RPC_URL;
+    getProvider(network?: string): ethers.providers.JsonRpcProvider {
+        let rpcUrl: string | undefined;
+        try {
+            rpcUrl = getNetworkConfigByType(network).url;
+        } catch {
+            rpcUrl = process.env.RPC_URL;
+        }
         if (!rpcUrl) {
             throw new Error(`No RPC_URL env variable was provided`);
         }
+        console.warn(`RPC_URL: ${rpcUrl}`);
```

Comment:
Please remove unused console logs. Also, it's recommended to use `console.log` or debugging mode for debugging